From c3c9c4e1c5069cad7e7aea2229319ed4be688c7a Mon Sep 17 00:00:00 2001
From: Ashwin Sekhar T K <asekhar@marvell.com>
Date: Tue, 14 Sep 2021 12:23:20 +0530
Subject: [PATCH 01/11] octeontx2-af: extend npa context reading capability

commit 455f55bf542fab97ef8e23f9f85fdf525727343f from
git@git.assembla.com:cavium/WindRiver.linux.git

Exporting the mbox handler used to read the npa AURA/POOL context.
This is used to honor npa context reading request made by user via
hw access driver.

Signed-off-by: Ashwin Sekhar T K <asekhar@marvell.com>
Change-Id: I9ae6647b12f7e5af54d7da32bec860698474b437
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/61208
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <sgoutham@marvell.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu_npa.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_npa.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_npa.c
index 6c239680a775..61593938bd5f 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_npa.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_npa.c
@@ -294,6 +294,7 @@ int rvu_mbox_handler_npa_aq_enq(struct rvu *rvu,
 	return rvu_npa_aq_enq_inst(rvu, req, rsp);
 }
 #endif
+EXPORT_SYMBOL(rvu_mbox_handler_npa_aq_enq);
 
 int rvu_mbox_handler_npa_hwctx_disable(struct rvu *rvu,
 				       struct hwctx_disable_req *req,
-- 
2.31.1

